web/client/reducers/users.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {
+    USERMANAGER_GETUSERS, USERMANAGER_EDIT_USER, USERMANAGER_EDIT_USER_DATA, USERMANAGER_UPDATE_USER, USERMANAGER_DELETE_USER,
+    USERMANAGER_GETGROUPS
+} = require('../actions/users');
+const assign = require('object-assign');
+function users(state = {
+    start: 0,
+    limit: 12
```

Comment:
12 is duplicated everywhere, can we make this a constant that we change in a single place?